src/ucp/core/ucp_ep.c

```diff
@@ -730,8 +730,9 @@ void ucp_ep_config_init(ucp_worker_h worker, ucp_ep_config_t *config)
                  * Isolating the 'size' yields the rndv_thresh.
                  * The used latency functions for eager_zcopy and rndv are also specified in
                  * the UCX wiki */
-                numerator = ((2 * iface_attr->overhead) + (4 * iface_attr->latency) +
-                             md_attr->reg_cost.overhead);
+                numerator = (2 * iface_attr->overhead) +
+                            (4 * ucp_tl_iface_latency(context, iface_attr)) +
+                            md_attr->reg_cost.overhead;
```

Comment:
thanks. just wanted to make sure that there isn't a significant change in the thresholds. in knem and cma also...